workflow-parser/src/action-v1.0.json

```diff
@@ -0,0 +1,556 @@
+{
```

Comment:
This file is basically a copy from the runner with the following changes:
- Added descriptions
- Added `-strict` definitions where we diverge and are more strict than the runner

This file is safe to copy back to the runner